guardian/actions-merge-release-changes-to-protected-branch
# Allow wildcard changes in additional changes input

## What does this change?

The `additional-changes` input allows a user to specify an object of files which they expect to change with an array of expected changes. In some scenarios, it may not be possible to know what the changes will look like so this PR allows the value `*` to be provided instead. This will accept any changes to the given file.

## How to test

Run a release using this option in the `additional-changes` object and see that it works as expected.

## How can we measure success?

Users can allow unknown changes to files.

## Have we considered potential risks?

This makes the action more permissive, potentially making it easier to automatically approve and merge unintended changes. As it is a user option and not a default, this risk should be acceptable.
